content/doc/developer/internationalization/i18n-source-code.adoc

```diff
@@ -25,9 +25,9 @@ Example.Description = A simple example for localization
 
 `src/main/resources/org/example/package/Messages_de.properties`:
 [source]
-Example.Description = Ein einfaches Beispiel f\u00fcr Lokalisierung
+Example.Description = Ein einfaches Beispiel für Lokalisierung
 
-The first file is the default localization for the _key_ `description` (used unless a more specific localization is available), while the second file provides the German localization.
+The first file is the default localization for the _key_ `description` (used unless a more specific localization is available), while the second file provides the German localization. Notice that UTF-8 is fully supported since Jenkins 2.361.1 (Java 11).
```

Comment:
more succinct and clear enough I think
```suggestion
The first file is the default localization for the _key_ `description` (used unless a more specific localization is available), while the second file provides the German localization. UTF-8 is supported since Jenkins 2.361.1 (Java 11).
```